core/src/main/java/hudson/tasks/ArtifactArchiver.java

```diff
@@ -206,6 +216,14 @@ public boolean isCaseSensitive() {
         this.caseSensitive = caseSensitive;
     }
 
+    public boolean isFlattenDirectories() {
+        return flattenDirectories;
+    }
+
+    @DataBoundSetter public final void setFlattenDirectories(boolean flattenDirectories) {
```

Comment:
Look up - I copy-pasted/edited it, and the other setters have it on the same line.